apps/src/componentLibrary/dropdown/simpleDropdown/simpleDropdown.module.scss

```diff
@@ -11,6 +11,23 @@
   gap: 0.125rem;
   margin: 0;
 
+  &.styleAsFormField {
+    width: 18.75rem;
```

Comment:
Curious why we're setting a hardcoded width when styling as form field? Should the consumer be managing the width instead?

Reply:
That's needed to make all the fields in deco have same default width. Consumer can still set custom width with custom className prop

Reply:
Gotcha, should this be a shared CSS constant then?

Reply:
Good point, updated. Thank you!